app/helpers/pages_helper.rb

```diff
@@ -30,16 +30,10 @@ def valid_hash?(data_type, data)
                   data['link'].is_a?(String)
     return false unless basic_check
 
-    data_type_check(data_type, data)
+    valid_data_type?(data_type, data)
   end
 
-  def data_type_check(data_type, data)
-    if data_type == 'partners'
-      data_type_check = data['image_link'].is_a?(String)
-    elsif data_type == 'resources'
-      data_type_check = data['tags'].is_a?(Array) &&
-                        data['languages'].is_a?(Array)
-    end
-    data_type_check
+  def valid_data_type?(data_type, data)
+    data['image_link'].is_a?(String) if data_type == 'partners'
```

Comment:
what about if data_type is `resources` why did we stop checking for array?

Reply:
The call to check `resources` data_type was never invoked anywhere in the application so decided to remove it, but I am open for any suggestions whether I should add another commit to add and implement check for `resources` data types

Reply:
I'm confused, I think it is necessary if you remove data_type_check(data_type, data) then we won't be able to see content for Partners and Resources

Reply:
It works as expected @julianguyen, I have tested it myself locally and it still works, let me request on slack for some few folks to test it out

Reply:
Oh yes you're right about this since we're using the Resources widget now. So only partners matters here. Thanks for clarifying